_includes/configuration/toolbar-persist.md

```diff
@@ -0,0 +1,15 @@
+## toolbar_persist
+
+Use this option to disable the automatic show and hide behaviour of the inline toolbar. Enabling this setting would, by default, always show the inline toolbar and not hide it whenever focus is lost.
```

Comment:
```suggestion
This option disables the automatic show and hide behavior of the inline toolbar and menu bar. Enable this option to always show the inline toolbar and menu bar and not hide them whenever focus is lost. Use this option to make the inline toolbar and menu bar persistent.
```

Reply:
`Enable this option to always show the toolbar and menu bar, and...` <- added comma? Overall wording feels a bit... repetitive?